MyBujo2.0/Business Rules/Utils/CoreDataManager.swift

```diff
@@ -110,8 +110,17 @@ class CoreDataManager: NSObject {
                     monthData.numberOfPhotos += photos.count
                 }
             }
+            if let feeling = day.feeling{
+                monthData.feelingsNumber = monthData.feelingsNumber.map({
+                    if $0.feeling == feeling{
+                        return ($0.feeling, $0.number + 1)
```

Comment:
You can use ternary operator:
```swift
return $0.feeling === feeling ? ($0.feeling, $0.number) : $0
```
Use the `===` if it is the same object reference